eQuiz/modules/eQuiz.Web/Areas/Admin/Scripts/review/group.filter.js

```diff
@@ -0,0 +1,21 @@
+﻿(function (angular) {
+    angular.module('equizModule').filter('groupFilter', GroupFilter);

```

Comment:
Use the following convention:

angular
.module('equizModule')
.filter('groupFilter', GroupFilter);